commcare_connect/opportunity/forms.py

```diff
@@ -1384,7 +1385,196 @@ def save(self, commit=True):
         instance.opportunity = self.opportunity
         instance.amount_usd = self.cleaned_data["amount_usd"]
         instance.exchange_rate = self.cleaned_data["exchange_rate"]
+        instance.service_delivery = self.invoice_type == PaymentInvoice.InvoiceType.service_delivery
 
         if commit:
             instance.save()
         return instance
+
+
+class AutomatedPaymentInvoiceForm(forms.ModelForm):
+    amount = forms.DecimalField(
+        label=_("Amount"),
+        decimal_places=2,
+    )
+    amount_usd = forms.DecimalField(
+        label=_("Amount (USD)"),
+        required=False,
+        decimal_places=2,
+    )
+    invoice_number = forms.CharField(
+        label=_("Invoice ID"),
+        required=False,
+        widget=forms.TextInput(attrs={"placeholder": _("Auto-generated on save")}),
+        help_text=_("This value is system-generated and unique."),
+    )
+    usd_currency = forms.BooleanField(
```

Comment:
What does "USD Currency" mean? This label and name are quite confusing.

Reply:
It specifies whether the invoice is specifying an amount using USD. The current production invoice form uses these terms so I left it as is.